AndroidHomeApplication/app/src/main/res/values/strings.xml

```diff
@@ -1,19 +1,11 @@
 <resources>
     <string name="app_name">AndroidHomeApplication</string>
     <string name="back">Back</string>
-    <string name="avengers_end_game">Avengers: \nEnd Game</string>
-    <string name="avengers_end_game_in_line">Avengers: End Game</string>
-    <string name="action_adventure_fantasy">Action, Adventure, Fantasy</string>
-    <string name="age">13+</string>
-    <string name="reviews">125 Reviews</string>
     <string name="storyline">Storyline</string>
-    <string name="storyline_description">After the devastating events of Avengers: Infinity War, the universe is in ruins. With the help of remaining allies, the Avengers assemble once more in order to reverse Thanos\' actions and restore balance to the universe.</string>
     <string name="cast">Cast</string>
-    <string name="robert_downey_jr">Robert Downey Jr.</string>
-    <string name="chris_evans">Chris Evans</string>
-    <string name="mark_ruffalo">Mark Ruffalo</string>
-    <string name="chris_hemsworth">Chris Hemsworth</string>
-    <string name="move_details_activity">Move Details Activity</string>
     <string name="movies_list">Movies list</string>
-    <string name="_137_min">137 MIN</string>
+
+    <string name="age_template">%1$d+</string>
+    <string name="reviews_template">%1$d Reviews</string>
+    <string name="time_template">%1$d MIN</string>
```

Comment:
sometimes it's OK to use caps in resources
but sometimes it's better to use 
textAllCaps for textView